src/Components/helpers.js

```diff
@@ -41,27 +51,62 @@ function formatDate(inputDate) {
   return `${day}-${month}-${year}`;
 }
 
-export const TextInputGroupBasic = () => {
-  const [value, setValue] = useState('');
+export const TextInputGroupBasic = ({
+  textValue,
+  setTextValue,
+  verifiedStatus,
+  isLoading,
+}) => {
   return (
-    <TextInputGroup>
-      <TextInputGroupMain
-        value={value}
-        onChange={(_event, value) => setValue(value)}
-      />
-    </TextInputGroup>
+    <Flex direction={{ default: 'column' }}>
+      <TextInputGroup>
+        <TextInputGroupMain
+          value={textValue}
+          onChange={(_event, value) => setTextValue(value)}
+        />
+        <TextInput
+          value={textValue}
+          type="text"
+          validated={!isLoading && verifiedStatus === 'error' && verifiedStatus}
+          customIcon={
+            (isLoading && <InProgressIcon />) ||
+            (!isLoading && verifiedStatus === 'success' && (
+              <Icon status="success">
+                <CheckCircleIcon />
+              </Icon>
+            ))
+          }
+          onChange={(_event, value) => setTextValue(value)}
+        />
+      </TextInputGroup>
+      {!isLoading && verifiedStatus === 'error' && (
+        <HelperText>
+          <HelperTextItem variant="error">
+            There was an error saving the note.
+          </HelperTextItem>
+        </HelperText>
+      )}
+    </Flex>
   );
 };
 
-export const DropdownBasic = ({ initialValue }) => {
+TextInputGroupBasic.propTypes = {
+  initialValue: PropTypes.string,
+  status: PropTypes.string,
+  id: PropTypes.number,
+};
+
+// eslint-disable-next-line react/prop-types
```

Comment:
Here as well. lets' not disable eslint rules, but try to comply with eslint.

Reply:
Good catch, I forgot I did this